STLYEGUIDE_SPRESS_UPDATE/src/content/assets/js/site.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file
+ * A JavaScript file for the site.
+ *
+ * Our JavaScript must be wrapped in a closure.
+ * @see https://drupal.org/node/1446420
+ * @see http://www.adequatelygood.com/2010/3/JavaScript-Module-Pattern-In-Depth
+ *
+ * @copyright Copyright 2016 Palantir.net
```

Comment:
@aCyborg copyright needs to be updated. Might as well make everything "2018" since we're so close.

Reply:
if we want to update to 2018, we will have to update here as well as sass partials.